### PR TITLE
Localizes in Hindi applications.md

### PR DESCRIPTION
**Pull request by CloudShiva7cr**
### Description
#Fixes **Localizes** content/en/docs/reference/glossary/applications.md to **Hindi** content/hi/docs/reference/glossary/applications.md

### CODE REVIEW

File: content/hi/docs/reference/glossary/applications.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: एप्लिकेशंस (Applications)
+id: applications
+date: 2019-05-12
+full_link:
+short_description: >
+ लेयर वह परत है जहाँ विभिन्न कंटेनराइज्ड एप्लिकेशंस रन करते है।
```
Comment (niranjandarshann) [inline content/hi/docs/reference/glossary/applications.md:7]:
```suggestion
 वह परत जहाँ विभिन्न कंटेनराइज्ड एप्लिकेशंस रन करते हैं।
```

File: content/hi/docs/reference/glossary/applications.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+aka:
+tags:
+- fundamental
+---
+ लेयर वह परत है जहाँ विभिन्न कंटेनराइज्ड एप्लिकेशंस रन करते है।
```
Comment (niranjandarshann) [inline content/hi/docs/reference/glossary/applications.md:12]:
```suggestion
 वह परत जहाँ विभिन्न कंटेनराइज्ड एप्लिकेशंस रन करते हैं।
```

**Review by niranjandarshann — COMMENTED** [review]
I have some suggestion for changes.

File: content/hi/docs/reference/glossary/applications.md (hunk 2 above)
Comment (CloudShiva7cr) [reply content/hi/docs/reference/glossary/applications.md]: @niranjandarshann  Sure! updated as per your suggestion, Thanks!!

Comment (CloudShiva7cr) [reply content/hi/docs/reference/glossary/applications.md]: @niranjandarshann Can you review this as well, I have made necessary changes as suggested by you!

**Review by niranjandarshann — COMMENTED** [review]
Changes are now easy to understand and in native simple Hindi to understand without changing the meaning of the sentence. Looks good to me .

**Comment by niranjandarshann** [comment]
Thanks For the PR 
/lgtm

**Review by dipesh-rawat — COMMENTED** [review]
/approve

**Pull request merged**